### ISSUE TITLE
Fail to load vader_lexicon.txt

### ISSUE THREAD

**Issue by ghost** [nltk/nltk#1296]
I'd like to use `SentimentIntensityAnalyzer`, but got such a error:

```
[Errno 2] No such file or directory: '/Users/yobichi/mc2/lib/python3.5/site-packages/nltk/sentiment/vader_lexicon.txt'
```

How can I fix it? Or where can I download it manually?

**Comment by afgiel** [comment]
[this line](https://github.com/nltk/nltk/blob/develop/nltk/sentiment/vader.py#L201) has been problematic for me as it makes it very difficult to manually point to the lexicon if its not within `nltk/sentiment/`

it seems https://github.com/nltk/nltk/commit/37a89c4a53ffc7723df9097c417cc00737f0494f attempted to ensure that `vader_lexicon.txt` was within `nltk/sentiment/` at distribution time but the version hasn't been bumped since that happened 

IMO, at the very least the loading of the lexicon should be performed with `nltk.data.load` so at least the standard `nltk_data` directories are checked before failing

**Comment by ewan-klein** [comment]
I didn't initially put the VADER lexicon into the nltk data repo because it's so closely tied to the VADER code -- it's not generically usable resource. However, given the continued difficulty with loading it, putting it in data is probably the right solution. I can try to do this at the weekend, but I'm more than happy if someone else does it before I do!

**Comment by alvations** [comment]
This issue is resolved in nltk/nltk_data#46 

```python
>>> from nltk.sentiment.vader import SentimentIntensityAnalyzer
>>> darth = SentimentIntensityAnalyzer()

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/liling.tan/git-stuff/nltk-alvas/nltk/sentiment/vader.py", line 200, in __init__
    self.lexicon_file = nltk.data.load(lexicon_file)
  File "/Users/liling.tan/git-stuff/nltk-alvas/nltk/data.py", line 833, in load
    opened_resource = _open(resource_url)
  File "/Users/liling.tan/git-stuff/nltk-alvas/nltk/data.py", line 951, in _open
    return find(path_, path + ['']).open()
  File "/Users/liling.tan/git-stuff/nltk-alvas/nltk/data.py", line 672, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource vader_lexicon not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('vader_lexicon')
  
  Searched in:
    - '/Users/liling.tan/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************

>>> nltk.download('vader_lexicon')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'nltk' is not defined

>>> import nltk
>>> nltk.download('vader_lexicon')
[nltk_data] Downloading package vader_lexicon to
[nltk_data]     /Users/liling.tan/nltk_data...
True

>>> darth = SentimentIntensityAnalyzer()
>>> darth.polarity_scores('This is a foo bar sentence')
{'neg': 0.0, 'neu': 0.755, 'pos': 0.245, 'compound': 0.0772}
```

**Comment by alvations** [comment]
Resolved in nltk/nltk_data#46

Closing issue in preparation for v3.2.5
